src/library_browser.js

```diff
@@ -331,7 +333,7 @@ var LibraryBrowser = {
           if (Browser.resizeCanvas) {
             Browser.setFullscreenCanvasSize();
           } else {
-            Browser.updateCanvasDimensions(canvas);
+            Browser.setCanvasSize(canvas);
```

Comment:
This doesn't look right since `setCanvasSize` takes a width and a height, not a canvas object.

Reply:
yes you are right. wth did I do? :(

Reply:
I profusely apologize for this oversight.

I just pushed a fix for it. Ironically the "wrong" code was still doing the "right" thing and that is the reason why I never saw it during my testing:

* calling `Browser.setCanvasSize(canvas)` ends up calling `Browser.updateCanvasDimensions(canvas, canvas);`
* and due to the test `if (wNative && hNative)` (`hNative` is `undefined`), it executes the `if`
* The normal call ends up calling `Browser.updateCanvasDimensions(canvas)` with both `wNative` and `hNative` undefined resulting in the exact same code path being triggered

The joys of javascript...

In any case, the code is now doing the "right" thing while being "right" as well :)